router/handle.go

```diff
@@ -305,6 +305,17 @@ func (rt *Handle) commitStatusList(workerJobStatuses *[]workerJobStatus) {
 		if err != nil {
 			rt.logger.Error("Unmarshal of job parameters failed. ", string(workerJobStatus.job.Parameters))
 		}
+		errorCode, err := strconv.Atoi(workerJobStatus.status.ErrorCode)
+		if err != nil {
+			errorCode = 200
+		}
```

Comment:
failure to parse error code would result in a zero error code which should be fine
```suggestion
		errorCode, _ := strconv.Atoi(workerJobStatus.status.ErrorCode)
```